### DISCUSSION TITLE
Bundlephobia is not shoing my npm package why?

### DISCUSSION THREAD

**Discussion by opensrc0** [npm/feedback#1095] [General]
I have published 2 npm package and one of them is showing in the Bundlephobia but the other one is not available in Bundlephobia.

----------------------------------------------------------------------------------------------------------------------------------

### 1. fe-theme Package is available in Bundlephobia

<img width="766" alt="Screenshot 2024-03-26 at 11 24 19 PM" src="https://github.com/npm/feedback/assets/6891544/18e9541f-0cd5-4906-bfd8-a1e90fcbb9b2">

----------------------------------------------------------------------------------------------------------------------------------

### 2. fe-pilot Package is not available in Bundlephobia
<img width="763" alt="Screenshot 2024-03-26 at 11 24 30 PM" src="https://github.com/npm/feedback/assets/6891544/70b28b1f-e742-4248-b14e-e4eb7804c75f">

**Comment by ljharb** [comment]
You'd have to ask bundlephobia, but the website has been having trouble for awhile, so it's probably related to that.

**Reply by opensrc0** [reply]
ok
